### DISCUSSION TITLE
Anybody found a solution to the app not launching on Mac OS?

### DISCUSSION THREAD

**Discussion by alkent76** [Anjok07/ultimatevocalremovergui#952] [General]
I posted this question in Q&A yesterday but nobody's responded so I'll try in here. I've read so many posts regarding UVR not launching on Mac. I've tried every solution but nothing has worked. My guess is maybe my machine is simply too old, but I live in hope. Has anybody found a solution??

I'm on a mid-2014 iMac running Big Sur 11.7.8 and have downloaded and installed Ultimate_Vocal_Remover_v5_6_MacOS_x86_64.dmg

Literally nothing happens when I try to launch.

**Comment by TacoShel** [comment]
Same here.. I got nothing.

**Reply by alkent76** [reply]
There's a few of us.

**Comment by Magnarius** [comment]
The same. MacBook Pro Early 2013, Catalina 10.15.7

**Comment by ambresakura** [comment]
Hi , I just made an account because I think i could help many of us who have problems lunching UVR on mac OS .
First, I have two MacBooks pro, one with Monterey which is like a version after Big Sure, macbook pro 2015ish i think , and second is a macbook pro like mid 2012 running Catalina. I had tried weeks ago to install UVR but wasn't able to lunch until yesterday / today on both of my macbook pro.

First instance of UVR i have succeed to run was on my Monterey one, what I did is that i saw a forum post and someone said to like right click on UVR application to show contents, then go on Mac os and click on UVR and it'll lunch a terminal, i did this and read the terminal and see somethings like , UVR script cannot lunch because "GOTHIC.ttf" missing, so i went on that directory to see if it was actually missing, and it was not, BUT, it was spelled "GOTHIC.TTF" (TTF in caps) (it's about a font) , so from there what I did is i renamed all 4 (i think?) Font files from "BLABLA.TTF" to "BLABLA.ttf" (ttf in lowercase) , then I relaunched the app and, Surprisingly, it did work and it was actually running.

So far so good it seams to work fine and all..


Then for the macbook pro 2012 all I did after many trials of installing and try running UVR 5,6 ,
I did go to the UVR gitHub and downloaded UVR 5.5 from september 29 , (you have to go on UVR GitHub, then find and go to a "releases" tab, then you go down to the 5.5 version, and then you go to "assets" tab , and finally you download the UVR 5.5 x86_64_9_29.dmg

So far so good it seams to work well too everything fine..
Hoped it helps few

Voilà.

**Reply by TacoShel** [reply]
UVR 5.5 x86_64_9_29.dmg works! No security warning. No Terminal. Thank you! https://github.com/Anjok07/ultimatevocalremovergui/releases

I still could not get 5.6 to work. My fonts from UVR were already lower case. I'm on Big Sur. As per instructions on version UVR 5.2, I downloaded Python 3.9 (an older version). Maybe that helped 5.5 to work.  

Cheers!

**Reply by Cromemco** [reply]
Thanks. I couldn't find the 5.5 links before because the exposed links in the 5.5 section were for 5.6. Didn't realize that I had to open the assets accordion. Downloaded the 9_29 file as TacoShel suggested and it seems to work again, although for some reason, the startup message says it's 5.6.

**Reply by xxVOXxx** [reply]
[ambresakura](https://github.com/ambresakura),

complete lifesaver.  UVR 5.5 x86_64_9_29.dmg now installs and works on OSX 10.14 Mojave for me after failing with 5.6 for the last few days.  Many thanks.

**Comment by alkent76** [comment]
Thank you all so much. Sometimes it's the simplest thing - install an earlier version! 

5.5 works and I didn't have to change font names or mess with Python.
